.github/workflows/stale-bot.yml

```diff
@@ -12,21 +12,21 @@ jobs:
     - uses: aws-actions/stale-issue-cleanup@v4
       with:
         issue-types: issues
-        ancient-issue-message: Hello 👋! It looks like this issue hasn’t been active in longer 
-          than five months. We encourage you to check if this is still an issue in the latest release. 
-          In the absence of more information, we will be closing this issue soon. 
-          If you find that this is still a problem, please feel free to provide a comment or upvote 
-          with a reaction on the initial post to prevent automatic closure. If the issue is already closed, 
+        ancient-issue-message: Hello 👋! It looks like this issue hasn’t been active in longer
+          than five months. We encourage you to check if this is still an issue in the latest release.
+          In the absence of more information, we will be closing this issue soon.
+          If you find that this is still a problem, please feel free to provide a comment or upvote
+          with a reaction on the initial post to prevent automatic closure. If the issue is already closed,
           please feel free to open a new one.
-        stale-issue-message: Hello 👋! It looks like this issue hasn’t been active in longer than two months. 
-          We encourage you to check if this is still an issue in the latest release. 
-          In the absence of more information, we will be closing this issue soon. 
-          If you find that this is still a problem, please feel free to provide a comment or upvote 
-          with a reaction on the initial post to prevent automatic closure. If the issue is already closed, 
```

Comment:
the pre-commit-hook auto-formatted those lines. Now that I see this, this might be wrong, as it trimmed all spaces in the message.